### ISSUE TITLE
Regression: eas build --platform ios fails on Windows

### ISSUE THREAD

**Issue by franciscosamuel** [expo/expo#33386]
### Summary

On Windows, 

`eas build --platform ios`

fails with 

`scheme 'AppName' does not exist, make sure it's marked as shared`

I dug Expo's source code and this is caused by the call to

`IOSConfig.BuildScheme.getArchiveBuildConfigurationForSchemeAsync`

The AppName.xcscheme file is not found because, in @expo/config-plugins, `readSchemeAsync` function:

```  
const allSchemePaths = findSchemePaths(projectRoot);
const re = new RegExp(`/${scheme}.xcscheme`, 'i');
const schemePath = allSchemePaths.find((i) => re.exec(i));
```

Under Windows, `findSchemePaths` returns the paths with a backslash ( \\ ) separator and the RegExp is looking for a forward slash ( / ) separator, so the file isn't found.

### Managed or bare?

bare

### Environment

expo-env-info 1.2.1 environment info:
    System:
      OS: Windows 11 10.0.22631
    Binaries:
      Node: 20.11.1 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.22 - C:\Program Files\nodejs\yarn.CMD
      npm: 10.2.4 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: AI-241.19072.14.2412.12360217
    npmPackages:
      @expo/webpack-config: ~19.0.1 => 19.0.1
      babel-preset-expo: ^10.0.0 => 10.0.1
      expo: ~50.0.3 => 50.0.13
      react: 18.2.0 => 18.2.0
      react-dom: 18.2.0 => 18.2.0
      react-native: 0.73.2 => 0.73.2
      react-native-web: ~0.19.10 => 0.19.10
    Expo Workflow: bare

WARNING: We recommend using PowerShell or Bash via WSL 2 for development with Expo CLI on Windows. You may encounter issues using cmd.exe.

Warning: Root-level "expo" object found. Ignoring extra keys in Expo config: "name", "displayName"
Learn more: https://expo.fyi/root-expo-object
- Running 14 checks on your project...
✔ Check package.json for common issues (0s)
- Running 14 checks on your project...
✔ Check Expo config for common issues (0s)
- Running 14 checks on your project...
✔ Check native tooling versions (0s)
- Running 14 checks on your project...
✔ Check for common project setup issues (0s)
- Running 14 checks on your project...
✖ Check dependencies for packages that should not be installed directly (0s)
- Running 14 checks on your project...
✔ Check for issues with Metro config (0s)
- Running 14 checks on your project...
✔ Check npm/ yarn versions (0s)
- Running 14 checks on your project...
✖ Check for app config fields that may not be synced in a non-CNG project (0s)
- Running 14 checks on your project...
✔ Check if the project meets version requirements for submission to app stores (0s)
- Running 14 checks on your project...
✔ Check Expo config (app.json/ app.config.js) schema (0s)
- Running 14 checks on your project...
✖ Check that packages match versions required by installed Expo SDK (2s)
- Running 14 checks on your project...
✔ Check for legacy global CLI installed locally (5s)
- Running 14 checks on your project...
✔ Check that native modules do not use incompatible support packages (5s)
- Running 14 checks on your project...
✔ Check that native modules use compatible support package versions for installed Expo SDK (5s)
- Running 14 checks on your project...

Detailed check results:

The package "expo-modules-core" should not be installed directly in your project. It is a dependency of other Expo packages, which will install it automatically as needed.
The package  "@types/react-native" should not be installed directly in your project, as types are included with the "react-native" package.
The package  "expo-permissions" was deprecated in SDK 41 and should be removed from your project because it may no longer compile on the latest SDK. It was replaced by permissions methods directly on modules, eg: MediaLibrary.requestPermissionsAsync().

This project contains native project folders but also has native configuration properties in app.json, indicating it is configured to use Prebuild. When the android/ios folders are present, EAS Build will not sync the following properties: orientation, icon, scheme, userInterfaceStyle, splash, ios, android, plugins.

Advice: Add '/android' and '/ios' to your .gitignore file if you intend to use CNG / Prebuild. Learn more: https://docs.expo.dev/workflow/prebuild/#usage-with-eas-build

2024-12-03T11:36:09.289Z expo:utils:variadic Parsed arguments (variadic: [], flags: { '--check': true }, extra: [])
2024-12-03T11:36:09.611Z expo:env No environment variables loaded from .env files.
2024-12-03T11:36:09.648Z expo:doctor:dependencies:bundledNativeModules Fetching bundled native modules from the server...
2024-12-03T11:36:09.841Z expo:doctor:dependencies:validate Checking dependencies for 50.0.0: {
  '@codler/react-native-keyboard-aware-scroll-view': '^2.0.1',
  '@expo-google-fonts/roboto': '^0.2.0',
  '@expo/match-media': '^0.4.0',
  '@expo/prebuild-config': '~6.7.0',
  '@expo/vector-icons': '^14.0.0',
  '@expo/webpack-config': '~19.0.1',
  '@react-native-async-storage/async-storage': '1.21.0',
  '@react-native-community/datetimepicker': '7.6.1',
  '@react-native-community/masked-view': '0.1.11',
  '@react-native-community/netinfo': '11.1.0',
  '@react-navigation/native': '^6.0.6',
  '@react-navigation/stack': '^6.0.11',
  '@types/react-dom': '~18.0.10',
  '@types/react-responsive': '^8.0.2',
  axios: '^1.3.4',
  'cross-env': '^7.0.3',
  crypto: 'npm:crypto-browserify',
  'exif-js': '^2.3.0',
  expo: '~50.0.3',
  'expo-asset': '^9.0.2',
  'expo-auth-session': '~5.4.0',
  'expo-build-properties': '~0.11.0',
  'expo-camera': '~14.0.2',
  'expo-constants': '~15.4.5',
  'expo-dev-client': '~3.3.7',
  'expo-dev-menu-interface': '^1.1.1',
  'expo-file-system': '~16.0.5',
  'expo-font': '~11.10.2',
  'expo-image': '~1.10.5',
  'expo-image-manipulator': '~11.8.0',
  'expo-image-picker': '~14.7.1',
  'expo-linear-gradient': '~12.7.1',
  'expo-linking': '~6.2.2',
  'expo-local-authentication': '~13.8.0',
  'expo-localization': '~14.8.3',
  'expo-mail-composer': '~12.7.1',
  'expo-modules-core': '~1.11.8',
  'expo-notifications': '~0.27.5',
  'expo-permissions': '14.4.0',
  'expo-random': '~13.6.0',
  'expo-screen-orientation': '~6.4.1',
  'expo-splash-screen': '~0.26.4',
  'expo-status-bar': '~1.11.1',
  'expo-system-ui': '~2.9.3',
  'expo-updates': '~0.24.8',
  'expo-web-browser': '~12.8.2',
  'expo-web-browserV2': 'npm:expo-web-browser@~10.0.3',
  'i18n-js': '^3.7.1',
  jest: '^29.2.1',
  'jest-expo': '~50.0.1',
  jsbarcode: '^3.11.5',
  'jwt-decode': '^3.1.2',
  'metro-config': '^0.80.5',
  moment: '^2.29.4',
  'native-base': '^3.3.4',
  'normalize-css-color': '^1.0.2',
  react: '18.2.0',
  'react-dom': '18.2.0',
  'react-native': '0.73.2',
  'react-native-animatable': '^1.3.3',
  'react-native-barcode-builder': '^2.0.0',
  'react-native-barcode-expo': '^1.1.1',
  'react-native-crypto': '^2.2.0',
  'react-native-gesture-handler': '~2.14.0',
  'react-native-gradle-plugin': '^0.71.19',
  'react-native-image-crop-picker': '^0.39.0',
  'react-native-logs': '^5.0.1',
  'react-native-qrcode-svg': '^6.0.6',
  'react-native-reanimated': '~3.6.0',
  'react-native-responsive-screen': '^1.4.2',
  'react-native-safe-area-context': '4.8.2',
  'react-native-screens': '~3.29.0',
  'react-native-signature-canvas': '^4.5.0',
  'react-native-svg': '14.1.0',
  'react-native-web': '~0.19.10',
  'react-native-webview': '13.6.4',
  'react-native-xml2js': '^1.0.3',
  'react-responsive': '^9.0.2',
  'react-scripts': '^5.0.1',
  'react-signature-canvas': '^1.0.3',
  'resize-observer-polyfill': '^1.5.1',
  stream: 'npm:stream-browserify',
  'styled-components': '^5.2.1',
  timers: '^0.1.1',
  xml2js: '^0.6.2',
  '@babel/core': '^7.21.3',
  '@babel/preset-env': '^7.20.2',
  '@babel/preset-react': '^7.18.6',
  '@react-native/js-polyfills': '^0.74.0',
  '@react-native/metro-babel-transformer': '^0.73.13',
  '@react-native/metro-config': '^0.73.3',
  '@types/jsbarcode': '^3.11.1',
  '@types/react': '~18.2.45',
  '@types/react-native': '~0.71.3',
  '@types/react-signature-canvas': '^1.0.1',
  '@types/styled-components': '^5.1.4',
  '@types/styled-components-react-native': '^5.2.0',
  'babel-loader': '^9.1.2',
  'babel-preset-expo': '^10.0.0',
  'date-fns': '^2.15.0',
  'icon-set-creator': '^1.2.6',
  'metro-runtime': '^0.80.5',
  tslint: '^6.1.3',
  typescript: '^5.3.0'
}
2024-12-03T11:36:09.842Z expo:doctor:dependencies:validate Comparing known versions: [
  '@expo/prebuild-config',
  '@expo/vector-icons',
  '@expo/webpack-config',
  '@react-native-async-storage/async-storage',
  '@react-native-community/datetimepicker',
  '@react-native-community/netinfo',
  '@types/react-dom',
  'expo',
  'expo-asset',
  'expo-auth-session',
  'expo-build-properties',
  'expo-camera',
  'expo-constants',
  'expo-dev-client',
  'expo-file-system',
  'expo-font',
  'expo-image',
  'expo-image-manipulator',
  'expo-image-picker',
  'expo-linear-gradient',
  'expo-linking',
  'expo-local-authentication',
  'expo-localization',
  'expo-mail-composer',
  'expo-modules-core',
  'expo-notifications',
  'expo-random',
  'expo-screen-orientation',
  'expo-splash-screen',
  'expo-status-bar',
  'expo-system-ui',
  'expo-updates',
  'expo-web-browser',
  'jest',
  'jest-expo',
  'metro-config',
  'react',
  'react-dom',
  'react-native',
  'react-native-gesture-handler',
  'react-native-reanimated',
  'react-native-safe-area-context',
  'react-native-screens',
  'react-native-svg',
  'react-native-web',
  'react-native-webview',
  '@babel/core',
  '@types/react',
  'babel-preset-expo',
  'typescript'
]
2024-12-03T11:36:09.842Z expo:doctor:dependencies:validate Skipping packages that cannot be versioned automatically: [
  '@codler/react-native-keyboard-aware-scroll-view',
  '@expo-google-fonts/roboto',
  '@expo/match-media',
  '@react-native-community/masked-view',
  '@react-navigation/native',
  '@react-navigation/stack',
  '@types/react-responsive',
  'axios',
  'cross-env',
  'crypto',
  'exif-js',
  'expo-dev-menu-interface',
  'expo-permissions',
  'expo-web-browserV2',
  'i18n-js',
  'jsbarcode',
  'jwt-decode',
  'moment',
  'native-base',
  'normalize-css-color',
  'react-native-animatable',
  'react-native-barcode-builder',
  'react-native-barcode-expo',
  'react-native-crypto',
  'react-native-gradle-plugin',
  'react-native-image-crop-picker',
  'react-native-logs',
  'react-native-qrcode-svg',
  'react-native-responsive-screen',
  'react-native-signature-canvas',
  'react-native-xml2js',
  'react-responsive',
  'react-scripts',
  'react-signature-canvas',
  'resize-observer-polyfill',
  'stream',
  'styled-components',
  'timers',
  'xml2js',
  '@babel/preset-env',
  '@babel/preset-react',
  '@react-native/js-polyfills',
  '@react-native/metro-babel-transformer',
  '@react-native/metro-config',
  '@types/jsbarcode',
  '@types/react-native',
  '@types/react-signature-canvas',
  '@types/styled-components',
  '@types/styled-components-react-native',
  'babel-loader',
  'date-fns',
  'icon-set-creator',
  'metro-runtime',
  'tslint'
]
2024-12-03T11:36:09.876Z expo:doctor:dependencies:validate Package versions: {
  '@expo/prebuild-config': '6.7.4',
  '@expo/vector-icons': '14.0.0',
  '@expo/webpack-config': '19.0.1',
  '@react-native-async-storage/async-storage': '1.21.0',
  '@react-native-community/datetimepicker': '7.6.1',
  '@react-native-community/netinfo': '11.1.0',
  '@types/react-dom': '18.0.11',
  expo: '50.0.13',
  'expo-asset': '9.0.2',
  'expo-auth-session': '5.4.0',
  'expo-build-properties': '0.11.1',
  'expo-camera': '14.0.6',
  'expo-constants': '15.4.5',
  'expo-dev-client': '3.3.10',
  'expo-file-system': '16.0.8',
  'expo-font': '11.10.3',
  'expo-image': '1.10.6',
  'expo-image-manipulator': '11.8.0',
  'expo-image-picker': '14.7.1',
  'expo-linear-gradient': '12.7.2',
  'expo-linking': '6.2.2',
  'expo-local-authentication': '13.8.0',
  'expo-localization': '14.8.3',
  'expo-mail-composer': '12.7.1',
  'expo-modules-core': '1.11.12',
  'expo-notifications': '0.27.6',
  'expo-random': '13.6.0',
  'expo-screen-orientation': '6.4.1',
  'expo-splash-screen': '0.26.4',
  'expo-status-bar': '1.11.1',
  'expo-system-ui': '2.9.3',
  'expo-updates': '0.24.12',
  'expo-web-browser': '12.8.2',
  jest: '29.7.0',
  'jest-expo': '50.0.4',
  'metro-config': '0.80.6',
  react: '18.2.0',
  'react-dom': '18.2.0',
  'react-native': '0.73.2',
  'react-native-gesture-handler': '2.14.1',
  'react-native-reanimated': '3.6.3',
  'react-native-safe-area-context': '4.8.2',
  'react-native-screens': '3.29.0',
  'react-native-svg': '14.1.0',
  'react-native-web': '0.19.10',
  'react-native-webview': '13.6.4',
  '@babel/core': '7.24.0',
  '@types/react': '18.2.66',
  'babel-preset-expo': '10.0.1',
  typescript: '5.4.2'
}
2024-12-03T11:36:09.882Z expo:doctor:dependencies:validate Incorrect dependencies: [
  {
    packageName: '@react-native-community/datetimepicker',
    packageType: 'dependencies',
    expectedVersionOrRange: '7.7.0',
    actualVersion: '7.6.1'
  },
  {
    packageName: 'expo',
    packageType: 'dependencies',
    expectedVersionOrRange: '~50.0.20',
    actualVersion: '50.0.13'
  },
  {
    packageName: 'expo-camera',
    packageType: 'dependencies',
    expectedVersionOrRange: '~14.1.3',
    actualVersion: '14.0.6'
  },
  {
    packageName: 'expo-constants',
    packageType: 'dependencies',
    expectedVersionOrRange: '~15.4.6',
    actualVersion: '15.4.5'
  },
  {
    packageName: 'expo-dev-client',
    packageType: 'dependencies',
    expectedVersionOrRange: '~3.3.12',
    actualVersion: '3.3.10'
  },
  {
    packageName: 'expo-file-system',
    packageType: 'dependencies',
    expectedVersionOrRange: '~16.0.9',
    actualVersion: '16.0.8'
  },
  {
    packageName: 'expo-localization',
    packageType: 'dependencies',
    expectedVersionOrRange: '~14.8.4',
    actualVersion: '14.8.3'
  },
  {
    packageName: 'expo-modules-core',
    packageType: 'dependencies',
    expectedVersionOrRange: '~1.11.14',
    actualVersion: '1.11.12'
  },
  {
    packageName: 'expo-notifications',
    packageType: 'dependencies',
    expectedVersionOrRange: '~0.27.8',
    actualVersion: '0.27.6'
  },
  {
    packageName: 'expo-splash-screen',
    packageType: 'dependencies',
    expectedVersionOrRange: '~0.26.5',
    actualVersion: '0.26.4'
  },
  {
    packageName: 'expo-system-ui',
    packageType: 'dependencies',
    expectedVersionOrRange: '~2.9.4',
    actualVersion: '2.9.3'
  },
  {
    packageName: 'expo-updates',
    packageType: 'dependencies',
    expectedVersionOrRange: '~0.24.13',
    actualVersion: '0.24.12'
  },
  {
    packageName: 'react-native',
    packageType: 'dependencies',
    expectedVersionOrRange: '0.73.6',
    actualVersion: '0.73.2'
  }
]
The following packages should be updated for best compatibility with the installed expo version:
  @react-native-community/datetimepicker@7.6.1 - expected version: 7.7.0
  expo@50.0.13 - expected version: ~50.0.20
  expo-camera@14.0.6 - expected version: ~14.1.3
  expo-constants@15.4.5 - expected version: ~15.4.6
  expo-dev-client@3.3.10 - expected version: ~3.3.12
  expo-file-system@16.0.8 - expected version: ~16.0.9
  expo-localization@14.8.3 - expected version: ~14.8.4
  expo-modules-core@1.11.12 - expected version: ~1.11.14
  expo-notifications@0.27.6 - expected version: ~0.27.8
  expo-splash-screen@0.26.4 - expected version: ~0.26.5
  expo-system-ui@2.9.3 - expected version: ~2.9.4
  expo-updates@0.24.12 - expected version: ~0.24.13
  react-native@0.73.2 - expected version: 0.73.6
Your project may not work correctly until you install the correct versions of the packages.
Found outdated dependencies
Advice: Use 'npx expo install --check' to review and upgrade your dependencies.

One or more checks failed, indicating possible issues with the project.

### Error output

eas build --profile preview --platform ios --no-wait --non-interactive

Using EAS CLI without version control system is not recommended, use this mode only if you know what you are doing.
Using EAS CLI without version control system is not recommended, use this mode only if you know what you are doing.
Using EAS CLI without version control system is not recommended, use this mode only if you know what you are doing.
Using EAS CLI without version control system is not recommended, use this mode only if you know what you are doing.
Using EAS CLI without version control system is not recommended, use this mode only if you know what you are doing.
The field "cli.appVersionSource" is not set, but it will be required in the future. Learn more: https://docs.expo.dev/build-reference/app-versions/
We detected that you are building for the "internal" distribution. Resolving the environment for environment variables used during the build to "preview". Set the "environment" field in the build profile if you want to specify the environment manually.
No environment variables with visibility "Plain text" and "Sensitive" found for the "preview" environment on EAS servers.

scheme 'Stand' does not exist, make sure it's marked as shared
Error: scheme 'Stand' does not exist, make sure it's marked as shared
    at readSchemeAsync (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\node_modules\@expo\config-plugins\build\ios\BuildScheme.js:87:11)
    at Object.getArchiveBuildConfigurationForSchemeAsync (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\node_modules\@expo\config-plugins\build\ios\BuildScheme.js:102:27)
    at resolveXcodeBuildContextAsync (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\build\project\ios\scheme.js:23:63)
    at async createIosContextAsync (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\build\build\ios\build.js:26:31)
    at async createBuildContextAsync (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\build\build\createContext.js:107:18)
    at async prepareAndStartBuildAsync (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\build\build\runBuildAndSubmit.js:215:22)
    at async runBuildAndSubmitAsync (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\build\build\runBuildAndSubmit.js:108:49)
    at async Build.runAsync (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\build\commands\build\index.js:43:9)
    at async Build.run (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\build\commandUtils\EasCommand.js:68:16)
    at async Build._run (C:\Users\Francisco\AppData\Roaming\nvm\v20.11.1\node_modules\eas-cli\node_modules\@oclif\core\lib\command.js:80:22)
    Error: build command failed.

### Reproducible demo or steps to reproduce from a blank project

Using the default new project, on Windows, run

`eas build --platform ios`

**Comment by szdziedzic** [comment]
It was fixed in https://github.com/expo/expo/pull/33468 and released in https://github.com/expo/expo/blob/sdk-52/packages/%40expo/config-plugins/CHANGELOG.md#9012--2024-12-05

I will bump these versions in the EAS CLI and release a new version today https://github.com/expo/eas-cli/pull/2744

**Comment by szdziedzic** [comment]
@franciscosamuel it should be fixed in `14.0.0`
